examples/Joints/Ingen/pinjoint.py

```diff
@@ -0,0 +1,16 @@
+from sympy import Symbol
+from pydy.bodies import Body
+from pydy.joints import PinJoint
+from pydy.joints_method import JointsMethod
+
+parent = Body('parent')
+child = Body('child')
+
+gravity = Symbol('gravity')
+child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
```

Comment:
get_masscenter is from RigidBody. I have not changed anything there. but Yeah, all the attributes in RigidBody are private and uses setters/getters. Should we change them?